app/views/budgets/edit.html.erb

```diff
@@ -1,4 +1,8 @@
 <h1><%= t(".title") %></h1>
-<%= render "form" %>
+<%= simple_form_for @budget, url: edit_budgets_path do |f| %>
```

Comment:
Why did you re-add the form here?
The form partial should still work

Reply:
It seems to be an issue with singular resources, you have to be explicit with the path

Reply:
You can pass a `url` in a locals hash. Have a look at what I've done with the users forms in the other branch...

Reply:
Thanks Elle, done in sha: 630ce3c